roles/edpm_neutron_dhcp/templates/wrappers/dnsmasq.j2

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+{% if edpm_neutron_dhcp_sidecar_debug -%}
+set -x
+{%- endif -%}
+
+ARGS="$@"
+
+IMAGE_NAME={{ edpm_neutron_dhcp_sidecar_dnsmasq_image_name }}
+NEUTRON_CONFIG_DIR={{ edpm_neutron_dhcp_agent_config_dir }}
+NEUTRON_LIB_DIR={{ edpm_neutron_dhcp_agent_lib_dir }}
+
+{% raw -%}
+# Extract the network namespace UUID from the command line args provided by
+# neutron. Typically of the form (with dnsmasq as an example):
+#
+# dnsmasq --no-hosts --no-resolv --except-interface=lo \
+#   --pid-file=/var/lib/neutron/dhcp/317716b8-919a-4a6f-8db1-78128ec3b100/pid  \
+#   --dhcp-hostsfile=/var/lib/neutron/dhcp/317716b8-919a-4a6f-8db1-78128ec3b100/host ...
+NETNS=$(ip netns identify)
+NAME=neutron-dnsmasq-${NETNS}
+
+CLI="nsenter --net=/run/netns/${NETNS} --preserve-credentials -m -t 1 podman"
+LOGGING="--log-driver k8s-file --log-opt path=/var/log/containers/stdouts/${NAME}.log"
+CMD='/usr/sbin/dnsmasq -k'
+
+LIST=$($CLI ps -a --filter name=neutron-dnsmasq- --format '{{.ID}}:{{.Names}}:{{.Status}}' | awk '{print $1}')
+
+# Find orphaned containers left for dead after its main process terminated by neutron parent process
+# FIXME(cjeanner): https://github.com/containers/libpod/issues/1703
+ORPHANS=$(printf "%s\n" "${LIST}" | grep -E ":(Exited|Created)")
+if [ -n "${ORPHANS}" ]; then
+    for orphant in $(printf "%s\n" "${ORPHANS}" | awk -F':' '{print $1}'); do
+        echo "Removing orphaned container ${orphant}"
+        # TODO(slaweq): script should at least log what error
+        # prevented to stop or rm orphaned container if there will be any error
+        $CLI stop ${orphant} || true
+        $CLI rm -f ${orphant} || true
+    done
+fi
+
+# If the NAME is already taken by a container, give it an unique name
+printf "%s\n" "${LIST}" | grep -q "${NAME}$" && NAME="${NAME}-$(date +%Y-%m-%d-%H%M%S-%N)"
+echo "Starting a new child container ${NAME}"
+$CLI run --detach ${LOGGING} \
+     -v ${NEUTRON_CONFIG_DIR}:/etc/neutron:ro \
+     -v /run/netns:/run/netns:shared \
```

Comment:
I wonder... why does dnsmasq need access to namespaces volume here? Same for /etc/neutron. Same for the other two mounts.